activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -535,7 +535,7 @@ def initialize_type_map(m = type_map)
           m.register_type "uuid", OID::Uuid.new
           m.register_type "xml", OID::Xml.new
           m.register_type "tsvector", OID::SpecializedString.new(:tsvector)
-          m.register_type "macaddr", OID::SpecializedString.new(:macaddr)
+          m.register_type "macaddr", OID::Macaddr.new(:macaddr)
```

Comment:
It looks like this was accidentally reintroduced, and is causing CI to fail:

```suggestion
          m.register_type "macaddr", OID::Macaddr.new
```

Reply:
Right you are. Sorry about that. Git was not my friend during that squash merge.